src/main/kotlin/logic/entities/ActionLog.kt

```diff
@@ -0,0 +1,15 @@
+package org.damascus.logic.entities
+
+import kotlinx.datetime.LocalDateTime
+import java.util.UUID
+
+data class ActionLog(
+    val userName: String,
+    val taskId: UUID,
```

Comment:
This should be null-able as some logs will be project related without a task, for example, Admin X created project Y.